libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -19,12 +19,23 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/**
+ * @file mqtt_lightweight.h
+ * @brief User-facing functions for serializing MQTT 3.1.1 packets. This header
+ * should be included for building a light-weight MQTT client bypassing the
```

Comment:
Nit: consistency with other usage
```suggestion
 * should be included for building a lightweight MQTT client bypassing the
```

Reply:
sure